CHANGELOG.md

```diff
@@ -1,9 +1,35 @@
-## 2021-01-08 - Release 2.0.1
+# Changelog
+
+## [2.0.2](https://github.com/voxpupuli/rspec-puppet-facts/tree/2.0.2) (2021-07-21)
+
+[Full Changelog](https://github.com/voxpupuli/rspec-puppet-facts/compare/2.0.0...2.0.2)
+
+**Implemented enhancements:**
+
+- Implement github action testing and codecov coverage reporting [\#129](https://github.com/voxpupuli/rspec-puppet-facts/pull/129) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- Cut a new release [\#120](https://github.com/voxpupuli/rspec-puppet-facts/issues/120)
+- Memoizing facts [\#114](https://github.com/voxpupuli/rspec-puppet-facts/issues/114)
+- Commit 21442e7 looks to introduce hard to debug behavior [\#97](https://github.com/voxpupuli/rspec-puppet-facts/issues/97)
+
+**Merged pull requests:**
+
+- Upgrade to GitHub-native Dependabot [\#126](https://github.com/voxpupuli/rspec-puppet-facts/pull/126) ([dependabot-preview[bot]](https://github.com/apps/dependabot-preview))
+- Version bump and CHANGELOG for v2.0.1 [\#125](https://github.com/voxpupuli/rspec-puppet-facts/pull/125) ([DavidS](https://github.com/DavidS))
+- Implement fact memoization [\#122](https://github.com/voxpupuli/rspec-puppet-facts/pull/122) ([ekohl](https://github.com/ekohl))
+
+## [2.0.0](https://github.com/voxpupuli/rspec-puppet-facts/tree/2.0.0) (2020-08-05)
+
+*the git tag for 2.0.1 is missing, it was probably created by hand*
```

Comment:
Can't we track this back? Perhaps @mcanevet still has it locally.

Reply:
the tag is missing, @mcanevet  doesn't have it available. I recreated it with
```
GIT_COMMITTER_DATE="2021-01-09 00:00" git tag -s 2.0.1 5d87fa0990ba9bf23ee127c1870cb41a1f2f3960
```